src/main/java/taf/YandexAccountPage.java

```diff
@@ -0,0 +1,45 @@
+package taf;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+public class YandexAccountPage {
+
+    private WebDriver driver;
+
+    private WebElement disk;
+
+    private List<WebElement> advertisment;
+
+    public YandexAccountPage(WebDriver driver) {
+        this.driver = driver;
+        PageFactory.initElements(driver, this);
+    }
+
+    public void waitYandexAccountPage(By by) {
+        new WebDriverWait(driver, 3)
+                .until(ExpectedConditions.visibilityOfElementLocated(by));
+    }
+
+    public YandexDiskPage switchToDisk() {
+
+        By diskButton = By.xpath("//a[@data-id='disk']");
+        disk = driver.findElement(diskButton);
+        disk.click();
+
+        By advertismentWindow = By.xpath("//button[@class='control button2 button2_view_classic button2_size_s button2_theme_clear dialog__close']");
```

Comment:
control
button2 
button2_view_classic 
button2_size_s 
button2_theme_clear 
dialog__close
there are several classes. If you want to use all of them (check is it really needs) via XPath ("//button[contains(@class,'control') and contains(@class,'button2'')] and ...")